18-additional-resources.Rmd

```diff
@@ -94,12 +96,24 @@ Bryan, J. & Hestor, J. *Happy git and github for the useR*. Retrieved from [http
 
 *Introduction to dplyr*. Retreived from [https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html](https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html)
 
+> `dplyr` is a R package that helps to manipulate and understand data. It provides simple “verbs”, functions that correspond to the most common data manipulation tasks, to help you translate your thoughts into code.Anyone with no programming experience can easily understand the verbs and use them for data analytics with ease.
```

Comment:
Minor, add a space before "Anyone"

Reply:
this also needs to be re-written to not use lingo from [the package](https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html)

Reply:
thank you for the feedback. I have updated the text.

Reply:
minor, add a space between "code." and "Anyone"?

Reply:
thank you for the feedback. I have updated the text.